src/test/javascript/spec/component/exam/participate/exam-exercise-update-highlighter.component.spec.ts

```diff
@@ -82,17 +79,16 @@ describe('ExamExerciseUpdateHighlighterComponent', () => {
 
                     component.exercise = programmingExerciseDummy;
                     const exerciseId = component.exercise.id!;
-                    const update = { exerciseId, problemStatement: updatedProblemStatementWithPlantUml };
+                    const update = { exerciseId, problemStatement: updatedProblemStatement };
 
                     fixture.detectChanges();
                     examExerciseIdAndProblemStatementSourceMock.next(update);
                 });
         });
 
-        it('should ignore plantuml diagrams in programming exercise problem statements for diff calculation', () => {
-            const result = component.exercise.problemStatement;
-            expect(result).toEqual(component.updatedProblemStatementWithHighlightedDifferences);
-            fixture.detectChanges();
+        it('should not highlight differences for programming exercise', () => {
+            expect(component.updatedProblemStatementWithHighlightedDifferencesHTML).toBeUndefined();
+            expect(component.updatedProblemStatementHTML).toBeUndefined();
```

Comment:
This test case is not super intuitive for me. Could you explain what's going on here?